src/utils/types.ts

```diff
@@ -77,6 +77,64 @@ export abstract class APIBase {
     }
 }
 
+export class RAWAPI extends APIBase {
+    protected jrpcVersion:string = "2.0";
+    protected rpcid = 1;
+    callMethod = async (method:string, params?:Array<object> | object, baseurl?:string):Promise<RequestResponseData> => {
```

Comment:
"callMethod" ceases to make sense for RAWAPI. Should just be "post", "put", "patch", "delete", or "update"...

actually wtf is RAWAPI actually RESTAPI? Let's turn RAWAPI into RESTAPI instead!!!